modules/tinymce/src/plugins/table/main/ts/ui/Buttons.ts

```diff
@@ -30,7 +30,7 @@ const addButtons = (editor: Editor, selectionTargets: SelectionTargets): void =>
 
   const cmd = (command: string) => () => editor.execCommand(command);
 
-  // TODO TINY-8172: unwind this before merging the feature branch
+  // TODO: TINY-8172 Unwind this when alternative solution is found
```

Comment:
I wouldn't worry about it, but just in case there is a small grammar error in these. It should be `when an alternative` (i.e. is missing `an`).